sdk/identity/azure-identity/azure/identity/constants.py

```diff
@@ -2,7 +2,9 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See LICENSE.txt in the project root for
 # license information.
-# --------------------------------------------------------------------------
+# -------------------------------------------------------------------------
+MSI_ENDPOINT = "MSI_ENDPOINT"
+MSI_SECRET = "MSI_SECRET"
```

Comment:
Out of curiosity why didn't you add this to the EnvironmentVariables class as with the others?

Reply:
Good question. I think past me had a reason for this that's irrelevant now so I've moved it into the class.